### ISSUE TITLE
KeyError while loading statsbomb open data

### ISSUE THREAD

**Issue by mepwang** [PySport/kloppy#161]
ubuntu 22.04
python 3.10 venv
kloppy 3.6.0

the following code cause a KeyError:
```python
from kloppy import statsbomb
dataset = statsbomb.load_open_data(match_id=7560,coordinates="statsbomb",event_types=["pass", "shot"])
```

trace stack:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/mep/da/env/lib/python3.10/site-packages/kloppy/_providers/statsbomb.py", line 62, in load_open_data
    return load(
  File "/home/mep/da/env/lib/python3.10/site-packages/kloppy/_providers/statsbomb.py", line 42, in load
    return deserializer.deserialize(
  File "/home/mep/da/env/lib/python3.10/site-packages/kloppy/infra/serializers/event/statsbomb/deserializer.py", line 718, in deserialize
    formation_change_event_kwargs = _parse_formation_change(
  File "/home/mep/da/env/lib/python3.10/site-packages/kloppy/infra/serializers/event/statsbomb/deserializer.py", line 408, in _parse_formation_change
    formation = formations[formation_id]
KeyError: 4132
```
same error occurred for statsbomb open data match id: 16029, 16136, 16149
please help here.

**Comment by mepwang** [comment]
4132 and 5122 formation exist?

**Comment by koenvo** [comment]
Hmm sorry for the delay here.

@MKlaasman, would you mind having a look at this one? I can remember you added several other formations before.

**Comment by MKlaasman** [comment]
Implemented a fix, by adding formations: 
-	Formation 4-1-3-2 (Game: 7560)
-	Formation 3-2-1-2-2 (Game: 16029)

However, there seems to be no problem on my side with match 16136 & 16149. It might have been another Error, @mepwang? If not, please provide the error messages and I will take a look. 

Moreover, the formations for these two matches don’t seem to have any Undefined formations. See below

- match 16136 - unique formations: 3-4-3 & 4-3-3
- match 16149 - unique formations: 3-5-2 & 4-3-3

**Comment by mepwang** [comment]
ubuntu22.04
pyenv&poetry
python 3.11 and kloppy 3.7.1
match 16136 & 16149 works now

**Comment by MKlaasman** [comment]
Thanks for your comment @mepwang.

Than it seems like this issue is fixed after merging #166. @koenvo / @JanVanHaaren

**Comment by koenvo** [comment]
Thanks for fixing!
